tasks/__init__.py

```diff
@@ -75,63 +75,41 @@ def test_module(ctx, module=None, params=None):
 
 
 @task
-def test_travis_on_prod(ctx):
+def test_selenium_on_prod(ctx):
     """
     Runs targeted prod smoke tests on the latest Chrome
     """
     flake(ctx)
     print('>>> Testing modules in "{}" in Chrome'.format('tests'))
-    test_travis_with_retries(
+    test_selenium_with_retries(
         ctx, 'Master', _get_test_file_list(), module=['-m', 'smoke_test']
     )
 
 
-@task
-def test_travis_part_one(ctx):
-    """Run first group of tests on the browser defined by TEST_BUILD."""
-    all_test_files = _get_test_file_list()
-    midpoint = len(all_test_files) // 2
-    file_list = all_test_files[:midpoint]
-    test_travis_with_retries(ctx, 'part one', file_list)
-
-
-@task
-def test_travis_part_two(ctx):
-    """Run second group of tests on the browser defined by TEST_BUILD."""
-    all_test_files = _get_test_file_list()
-    midpoint = len(all_test_files) // 2
-    file_list = all_test_files[midpoint:]
-    test_travis_with_retries(ctx, 'part two', file_list)
-
-
 def _get_test_file_list():
     all_test_files = glob.glob('tests/test_*.py')
     all_test_files.sort()
     return all_test_files
 
 
 @task
-def test_travis_with_retries(ctx, partition_name, file_list, module=None):
+def test_selenium_with_retries(ctx, file_list, module=None):
```

Comment:
Where is the file_list parameter being populated now when you are directly invoking this function from the selenium_staging_tests.yml file?

Reply:
:+1